config/dataset/ccmeo_datacube_dataset.yaml

```diff
@@ -7,7 +7,7 @@ dataset:
   use_stratification: False
   train_val_percent: {'trn':0.7, 'val':0.3, 'tst':0}
   raw_data_csv: ${general.raw_data_csv}
-  raw_data_dir: ${general.raw_data_dir}
+  raw_data_dir: data  #${general.raw_data_dir}
```

Comment:
Why hardcode directory here?
Shouldn't remain configrable through the main config file, here: https://github.com/CharlesAuthier/geo-deep-learning/blob/test/222-stac-item-input/config/ccmeo_production.yaml#L32 ?